demo/nexmark/README.md

```diff
@@ -0,0 +1,81 @@
+# Nexmark
+
+These instructions will allow you to run the Nexmark benchmark using
```

Comment:
The queries are written in SQL.
Currently only a few of the queries from the benchmark are executed.
Notice that there is a separate Nexmark benchmark where more queries are implemented in Rust.